scottohara.me.txt

```diff
@@ -0,0 +1,5 @@
+title: //h1
+author: "Scott O'Hara"
```

Comment:
Wait, is it working? Shouldn't this be only an XPath?

Reply:
Yep, this is working event if I never thought about it.